src/zimdump.cpp

```diff
@@ -347,18 +327,18 @@ void ZimDumper::dumpFiles(const std::string& directory)
     {
         auto redirectArticle = it->getRedirectArticle();
         std::string redirectUrl = redirectArticle.getUrl();
-        if (redirectArticle.getMimeType() == "text/html")
+        if (symlinkdump)
+        {
+            symlink(redirectUrl.c_str(), f.c_str());
+        }
+        else
```

Comment:
We must do a symlink even if symlinkdump is false for other content than html:
- Symlinkdump [default]
  => symlink for all content.
- HTMLRedirect:
  => HTML redirect for html content 
  => Symlink for other content.